dist/linux/install.sh

```diff
@@ -99,19 +99,6 @@ if [ $? -eq 0 ]; then
     exit 1
 fi
 
-# if secure efi is not enabled, require tboot has been installed.  Note: This must
-# be done manually until RHEL 8.3 (a manual patch is required).
-bootctl status 2> /dev/null | grep 'Secure Boot: disabled' > /dev/null
-if [ $? -eq 0 ]; then
-    SUEFI_ENABLED="false"
-    
-    rpm -qa | grep ${TBOOT_DEPENDENCY} >/dev/null
```

Comment:
remove tboot package from installer

Reply:
@ddhawal, This code is already removed. tboot reference is there in case of SUEFI enabled to check both SUEFI and tboot are not available at the same time. I think that check is needed.

Reply:
if tboot itself is removed then check is not required

Reply:
@ddhawal, Addressed the comment.